libs/examples/src/select/select-custom-value-template-example/select-custom-value-template-example.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'dt-example-select-custom-value-template-example',
+  templateUrl: './select-custom-value-template-example.html',
+  styles: [
+    '::ng-deep .dt-option-label, ::ng-deep .dt-select-trigger .dt-select-value { display: flex !important; align-items: center; }',
```

Comment:
Please remove the !important override and use a more specific selector instead. Since this is an example people tend to copy those we should avoid this here.